allthekernels.py

```diff
@@ -118,10 +118,13 @@ def start_kernel(self, name):
             connection_file=cf,
         )
         manager.start_kernel()
-        self.kernels[name] = kernel = KernelProxy(
+        self.kernels[name] = kernel_client = KernelProxy(
```

Comment:
`kernel_client` is not used.
```suggestion
        self.kernels[name] = KernelProxy(
```

Reply:
This has been fixed with commit https://github.com/minrk/allthekernels/commit/08863d03a325475b17bba85cac48b1492ebba721